### ISSUE TITLE
Missing cealign module

### ISSUE THREAD

**Issue by totucuong** [biopython/biopython#4019]
### Setup

I am reporting a problem with Biopython version, Python version, and operating
system as follows:

```python
import sys; print(sys.version)
import platform; print(platform.python_implementation()); print(platform.platform())
import Bio; print(Bio.__version__)
```

CPython
Darwin-21.5.0-x86_64-i386-64bit
1.79
### Expected behaviour

cealign should be imported using:

```import Bio.PDB.cealign```

### Actual behaviour
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'Bio.PDB.cealign'
### Steps to reproduce

Just import the cealign module

**Comment by JoaoRodrigues** [comment]
The cealign module will be available in version 1.80 and onwards. In the
meantime, you can clone and install the Biopython git repository to access
the new module.

A quarta, 3/08/2022, 05:29, To Tu Cuong ***@***.***> escreveu:

**Comment by totucuong** [comment]
Thanks @JoaoRodrigues. It makes sense now :)
